cypress/utils/plugins/anomaly-detection-dashboards-plugin/constants.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { BASE_PATH } from '../../base_constants';
```

Comment:
Can you update for [gantt charts](https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/cypress/integration/plugins/gantt-chart-dashboards/gantt_ui.spec.js#L8) as well?

Reply:
So outside the utils folder, constants should all be import from `'../../../utils/constants'`